src/app/models/user.model.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright 2020 Google LLC
+ */
+export interface User {
+  uid: string | undefined;
+  displayName?: string | null;
```

Comment:
Why do we need ` | null`? We already have `?:` here which allows `displayName` to be `undefined`. Not sure I see we need to make the user deal with both `undefined` and `null` here.

Reply:
I added it because it can come back as null from Firebase but thinking about it more it makes more sense to just check if it returns null and only add it if it does exist, I'll implement that.